ocs_ci/ocs/resources/catalog_source.py

```diff
@@ -180,3 +181,21 @@ def enable_specific_source(source_name):
     )
     logger.info(f"Waiting 20 seconds after enabling source: {source_name}")
     sleep(20)
+
+
+def get_odf_tag_from_redhat_catsrc():
+    """
+    Get the ODF tag from the default redhat-operators Catalog Source
+
+    Returns:
+        str: ODF tag from redhat-operators Catalog Source
+    """
+    from ocs_ci.ocs.ocp import OCP
+
+    catsrc_data = OCP(
+        kind=constants.CATSRC,
+        namespace=constants.MARKETPLACE_NAMESPACE,
+        resource_name="redhat-operators",
+    ).get()
+    regestry_image = catsrc_data.get("spec").get("image")
+    return regestry_image.split(":")[-1]
```

Comment:
```suggestion
    registry_image = catsrc_data.get("spec").get("image")
    return registry_image.split(":")[-1]
```

Also I'm not sure, if it is applicable for the places, where this function is used, but this will not work correctly for GA version of ODF.

Reply:
addressed grammar and added your concerns to the ticket https://github.com/red-hat-storage/ocs-ci/issues/10279
thanks